sway-book/src/documentation/language/program-types/predicate.md

```diff
@@ -0,0 +1,17 @@
+# Predicates
+
+A predicate is an executable that represents a UTXO spending condition, such as a multisig predicate, which has restrictions on the VM instructions that can be used (e.g. no jumps).
```

Comment:
It would be accurate to say that the restrictions are that a predicate cannot be stateful, rather than using jumps as an example

Reply:
There is a sentence below that mentions the state. Does that address this concern?